apps/patient/src/views/Pharmacy.tsx

```diff
@@ -382,28 +391,19 @@ export const Pharmacy = () => {
               pharmacy: selectedPharmacy
             });
 
-            navigate(`/status?orderId=${order.id}&token=${token}&type=${type}`);
+            return navigate(`/status?orderId=${order.id}&token=${token}&type=${type}`);
           }, 1000);
         } else {
-          toast({
-            title: isReroute ? 'Unable to reroute order' : 'Unable to submit pharmacy selection',
-            description: 'Please refresh and try again',
-            ...TOAST_CONFIG.ERROR
-          });
+          showToastError();
         }
         setSubmitting(false);
       }, 1000);
     } catch (error) {
-      toast({
-        title: isReroute ? 'Unable to reroute order' : 'Unable to submit pharmacy selection',
-        description: 'Please refresh and try again',
-        ...TOAST_CONFIG.ERROR
-      });
-
+      showToastError();
       setSubmitting(false);
-
       console.error(JSON.stringify(error, undefined, 2));
     }
+    navigate(`/status?orderId=${order.id}&token=${token}`);
```

Comment:
in this scenario, maybe something else has happened since they tried changing so they no longer can make a change. We should kick them out of this view when we show them the error toast